types/node/http.d.ts

```diff
@@ -395,11 +395,85 @@ declare module "http" {
         destroy(): void;
     }
 
-    const METHODS: string[];
+    const METHODS: [
+        'ACL',        'BIND',        'CHECKOUT',
+        'CONNECT',    'COPY',        'DELETE',
+        'GET',        'HEAD',        'LINK',
+        'LOCK',       'M-SEARCH',    'MERGE',
+        'MKACTIVITY', 'MKCALENDAR',  'MKCOL',
+        'MOVE',       'NOTIFY',      'OPTIONS',
+        'PATCH',      'POST',        'PRI',
+        'PROPFIND',   'PROPPATCH',   'PURGE',
+        'PUT',        'REBIND',      'REPORT',
+        'SEARCH',     'SOURCE',      'SUBSCRIBE',
+        'TRACE',      'UNBIND',      'UNLINK',
+        'UNLOCK',     'UNSUBSCRIBE'
+    ];
 
     const STATUS_CODES: {
-        [errorCode: number]: string | undefined;
```

Comment:
This doesn't seem good - it seems entirely likely there is code today that gets a `number` and looks up into this table, and this change will break that code.

Reply:
Removed changes to `STATUS_CODES`.

Don't think it's possible to keep it a `number` compatible and have something like this:
```ts
type HTTPStatusCode = keyof typeof http.STATUS_CODES;
```
At least I don't know how :thinking:

Reply:
@qwelias, this comment is about a change that you removed, right?

Reply:
It appears that way—@qwelias not a big deal, but since we have a weekly rotation for DefinitelyTyped PR review, avoiding force-pushes can help us figure out whether threads like this have actually been resolved or not 👍

Reply:
Apologies for confusion :+1: